reana_workflow_controller/config.py

```diff
@@ -117,6 +117,31 @@
 ]
 """Common to all workflow engines environment variables."""
 
+
+def _env_vars_dict_to_k8s_list(env_vars):
+    return [{"name": name, "value": value} for name, value in env_vars.items()]
```

Comment:
Maybe we could move this function to the top of the file, just to avoid having it in the middle of the variables declaration?